scripts/linker/analyze_classification_links.py

```diff
@@ -0,0 +1,181 @@
+"""
+This script creates a summary and graphs derived from the classification results.
+
+It needs to be executed inside the directory which contains the classification links files
+"""
+
+from glob import glob
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+sns.set(rc={"lines.linewidth": 0.5})
+
+df = pd.DataFrame(columns=[
+    "Catalog", "Entity", "Model", "Count", "Mean", "STD",
+    "Min", "25%", "50%", "75%", "Max"
+])
+
+files = glob("*_links.csv.gz")
+all_links = pd.DataFrame(columns=[
+    "Catalog",
+    "Entity",
+    "Model",
+    "QID",
+    "TID",
+    "Prediction"
+])
+
+non_binary_classifiers = [
```

Comment:
yes it might be a bit confusing. I've renamed this list to _continuous_classifiers_ in b7922195e04a1390c74413c9de3b5cc2a53e3543 . The distinction is that these classifiers give as a result a prediction which is a continuous value (instead of just the label `[0,1]`)